src/components/people/PersonCard/index.tsx

```diff
@@ -32,9 +33,9 @@ export default ({
     showJobTitle,
 }: PersonCardProps) => {
     const [currentPerson, setCurrentPerson] = useState<PersonDetails>();
-    const { error, personDetails } = personId
+    const { isFetching, error, personDetails } = personId
         ? usePersonDetails(personId)
-        : { error: null, personDetails: person };
+        : { isFetching: true, error: null, personDetails: person };
```

Comment:
We should probably have isFetching as an optional prop (isFetchingPerson?: boolean) and use this as isFetching when using the person prop instead of the personId prop

Reply:
Fixed